### PR TITLE
fix: do not break `calc` with single var

### PR DESCRIPTION
**Pull request by evilebottnawi**
Original issue https://github.com/cssnano/cssnano/issues/725

### CODE REVIEW

**Comment by andyjansson** [comment]
This fix is done in the wrong place. Adjust output in the stringifier, not in the transformer.

**Comment by evilebottnawi** [comment]
@andyjansson disagree, `calc(env(safe-*))` should be simplify as `env(safe-*)`

**Comment by evilebottnawi** [comment]
@andyjansson I understand correctly that you don't merge this because you don't like solutions and `we should not parse when it is unnecessary (it is increase build/minification time).` is not enough for you?

**Comment by evilebottnawi** [comment]
Don't merge i find other edge case `calc(calc(var(--foo)))`, need fix (WIP)

File: src/lib/stringifier.js [hunk 1]
```diff
@@ -1,69 +1,73 @@
...
+    // if the warnWhenCannotResolve option is on, inform the user that the calc
+    // expression could not be resolved to a single value
+    if (options.warnWhenCannotResolve) {
+      result.warn(
+        "Could not reduce expression: " + originalValue,
+        { plugin: 'postcss-calc', node: item });
+    }
+  }
+  return str;
+}
```
Comment (evilebottnawi) [inline src/lib/stringifier.js]: Already somebody use `CRLF` in code :confused:

File: src/lib/stringifier.js [hunk 2]
```diff
@@ -1,69 +1,73 @@
...
+    originalValue,
+    options,
+    result,
+    item
+  ) {
+  let str = stringify(node, options.precision);
+
+  const shouldPrintCalc =
+    node.type === "MathExpression" ||
+    (node.type === "Function" && node.value.toLowerCase().slice(0, 4) === "var(");
```
Comment (evilebottnawi) [inline src/lib/stringifier.js]: Only this was added

**Comment by evilebottnawi** [comment]
Ready for merge and review

**Pull request merged**